pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -52,6 +52,9 @@ type capability struct {
 	encoders       map[string]types.Encoder
 
 	callbackChannelBufferSize int
+
+	registeredWorkflows   map[string]struct{}
```

Comment:
Nit: I often find it easier to use a map[string]bool since you can then use the following shortform to test presence:

`if registeredWorkflows["aKey"] ...` rather than having to check the presence operator via `_, ok := registeredWorkflows["aKey"]`